group_vars/all/netbox_prefixes.yaml

```diff
@@ -0,0 +1,85 @@
+netbox_prefixes:
+  - prefix: 10.0.0.0/19
+    vlan:
+      name: MOC IPMI
+    description: "MOCs main IPMI network"
+  - prefix: 128.52.60.0/22
+    vlan:
+      name: CSAIL 10
+    description: "Public IPs from CSAIL"
+  - prefix: 192.12.185.0/24
+    vlan:
+      name: BU Public
+  - prefix: 129.10.5.0/24
+    vlan:
+      name: NEU Public
+  - prefix: 172.16.0.0/19
+    vlan:
+      name: Foreman
+  - prefix: 172.16.32.0/19
+    vlan:
+      name: Openstack Internal API
+  - prefix: 172.16.64.0/19
+    vlan:
+      name: Openstack Tenant Network
+  - prefix: 172.16.96.0/19
+    vlan:
+      name: HIL Internet
+  - prefix: 192.168.0.0/24
+    vlan:
+      name: oVirt Gluster
+  - prefix: 10.1.0.0/24
+    vlan:
+      name: OCT Switch Management
+  - prefix: 10.253.0.0/23
+    vlan:
+      name: Zero Cluster NFS
```

Comment:
Technically not just the zero cluster. "OpenShift nfs", maybe?

Reply:
done